common/country_types/hfe_country_types.txt

```diff
@@ -115,84 +122,117 @@ human_fallen_empire = {
+		
 		exclusive_diplomacy_module = {
 			can_receive = {
 				action_declare_war
+				action_offer_peace_diplomacy_view
 				action_offer_peace
 				action_make_rival
 				action_end_rivalry
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_insult
+				action_support_independence
 			}
 			can_send = {
 				action_declare_war
+				action_offer_peace
+				action_offer_peace_diplomacy_view
 				action_make_rival
 				action_end_rivalry
 				action_offer_peace
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_open_borders
 				action_close_borders
+				action_insult
+				action_support_independence
+#				action_ask_to_become_testing_arry = yes
 			}
 		}
 	}
+	resources = {
+		category = country_base
+		produces = {
+			energy = 100
+			minerals = 100
+			food = 100
+			alloys = 100
+			consumer_goods = 100
+			physics_research = 100
+			society_research = 100
+			engineering_research = 100
+			influence = 10
+			unity = 5
+		}
+	}
 }
 
 hfe_drone = {
 	has_difficulty_bonuses = yes
 	counts_for_victory = no
-	relations = yes # без этого не строит корабли лул
+	relations = yes # ??? ????? ?? ?????? ??????? ???
 	observable = yes
-	needs_survey = yes
-
+	needs_survey = no
+	has_complete_sensor = yes
 	playable = no
 	destroys_starbases = yes
 	share_survey_data = no
-	is_space_critter = yes
-	blocks_piracy = yes
+	is_space_critter = yes # This fixes a bug where the Drones can capture Habtiable Planets and have a primary ruling species
+	can_own_empty_planets = no
+	blocks_piracy = no
 	government = no
 	has_capital = no
 	enforces_borders = no
-	share_communications = no
-	can_have_their_comms_shared = no
+	orbital_bombardment = yes
+	attack_unarmed_orbital_stations = yes
+	share_communications = yes
+	can_have_their_comms_shared = yes
 	mid_game_crisis = yes
 	sub_title = personality_hfe_personality_drone
+	sub_title_desc = personality_hfe_personality_drone_desc
 
 	# targets = {
 	# 	hfe_drone
 	# }
 
 	ai = {
 		enabled = yes
-		#declare_war = yes
-		#min_navy_for_wars = 0.5
-		colonizer = yes
-		internal_policies = yes # нужно для постройки станций
+#		declare_war = yes
+#		min_navy_for_wars = 0.5
+		colonizer = yes # Space drones can't colonize a planet, but they can build Outposts!
+		internal_policies = yes # ????? ??? ????????? ???????
 		construction = yes
 		tech = yes
 		government = no
 		traditions = no
-		#can_use_market = yes
+		starbase_upgrade = yes
+		can_use_market = yes
 		modules = {
 			military_minister_module
-			interior_minister_module # не убирать ни в коем случае, нужно для постройки кораблей
-			foreign_minister_module # а без этого не упгрейдит аутпосты
+			interior_minister_module # ?? ??????? ?? ? ???? ??????, ????? ??? ????????? ????????
+			foreign_minister_module # ? ??? ????? ?? ????????? ????????
 		}
 		ship_data = {
 			# need any designable ship size
 			constructor = {
-				min = 3
-				max = 12
+				min = 4
+				max = 25
+			}
+			battleship = {
```

Comment:
It had not battleship class for a reason

Reply:
Okay, I will change it the next time I get on.

Reply:
Okay, done